vimeo-networking/src/main/java/com/vimeo/networking/VimeoService.java

```diff
@@ -133,9 +150,26 @@
      */
     // <editor-fold desc="Editing">
     @PATCH
-    Call<Object> edit(@Header("Authorization") String authHeader, @Url String uri,
+    Call<Object> edit(@Header("Authorization") String authHeader,
```

Comment:
Fantastic suggestion. I will add it to my follow up list of tickets to be created.